docs/source/examples/demo_metrics_correlation.ipynb

```diff
@@ -0,0 +1,496 @@
AISjcG4zgHfXZKRmKi1CvdxkJHCUO6++Fp07i+bRsnZs2igHPSAsFTxY5PPmFFr14s07czfRROzp4tLT8IDeXAlCnER0Q88nlLNCW8Jj1h/27QarGvVsNIoAsEgtJJgUX666+/LrX0atiwIREREZw4cYKbN2/St2/ffJ8nMzOTU6dO0a7dQ/dJpVJJu3btOGKStmigefPmnDp1ShLlYWFhbN68mZdeeinH62RkZJCYmGj0EAgEAkHpwmAc52gQ6UlJkkgvK+v8YKXvkmAwjjs9d67ZOQSCZxHD38KNPXse+Vyxly5Jyyv69GH355+z6IUXHvm8JRmFUl+Tri2Z7u6GenSR6i4QPB0UWKQfPHjQ6LmDgwMNGjTItzmWgXv37qFWq6U+1AZ8fX2JioqyeEz//v2ZPHkyLVu2xNramkqVKtGmTRs+++yzHK8zdepUXF1dpUf58uULNE6BjsDAQLRarUh1FwgExYLaQiQ924JId9b7Q5yeO5fkqCgSb96UtgkzOcGzjKaIIsAatZq0+/el54bluEK0WS1NKJT6lPoSGEnXqtUk7NsBgFvbjsU8GoFAUBQUWKS/8MILBAUF8dlnn3FJNpP6JNi7dy9Tpkzht99+4/Tp06xevZpNmzbx1Vdf5XjMuHHjSEhIkB43ZTdsAoFAICgdGKLgDvoJ4UxZJN2l3MPUznJ6wZ4cFcW6oUONziHasgmeZYqqv3fagwclulf4Y6ME90lPvXSe7Af3UTo549SwSd4HCASCEk+BRfqdO3f48MMP2bdvH7Vq1aJevXr88MMP3Lp1q0Dn8fLyQqVSER0dbbQ+OjoaPz8/i8eMHz+egQMHMnz4cGrXrk2PHj2YMmUKU6dORZPDl6atrS0uLi5GD4FAIBCULnKrSbd2cODdkBBe37YNv/r1pWNCt241OocQ6YJnmaISlyn6jiPPGoYWbJTAdPfEo7osV5fGzVFaWxfzaAQCQVFQYJHu5eXFyJEjOXToENevX6d379789ddfBAYG8kIB6pFsbGxo2LAhu3btktZpNBp27dpFM5N+twZSU1PN2k2pDH0rhSGQQCAQPLUY0t0NNekpMTE8CA0FdCLdq3p1KnXo8PBG2gJCpAueaYroPulZFemGFmxadUkU6QcAcG7SsphHIhAIiopHarAcFBTEp59+yrfffkvt2rXZt29fgY4fM2YMc+fO5a+//iIkJIR33nmHlJQUhupTFAcNGsS4ceOk/bt27crvv//O0qVLCQ8PZ8eOHYwfP56uXbtKYl0gEAgETx9SurtepANGIt1Ag+HDqSgzJJUjRLpA8Og8qyJdmgAsYenuWrWapOM6w2WXZs8X82gEAkFRYVXYAw8dOsSSJUtYuXIl6enpdOvWjalTpxboHH379iU2NpYJEyYQFRVFvXr12Lp1q2QmFxkZaRQ5/+KLL1AoFHzxxRfcvn0bb29vunbtyjfffFPYlyEQCASCUkC2SSRdjlyk2zo78/r27Xzn7k5GQoLRfhnCOE4geGRyE+lajQaF8pHiPyUXQ016CTOOSz57EnViPCoXNxxr1i3u4QgEgiKiwCJ93LhxLF26lDt37tC+fXt++uknunXrhoPsJqkgjBw5kpEjR1rctnfvXuPBWlkxceJEJk6cWKhrCQQCgaB0Yoik2zg74+jrS4rMz8Ta5PdHoVDgVa0at/XtOhVKJVqNxmIkPTMlBRtHx8c4coHg6cIg0l0DAkgw6Y2emZyM7VPq/WNowVbSatINrddcW7+IwqrQsTeBQFDCKPB05/79+/n444+5ffs2Gzdu5LXXXiu0QBeUHBYuXIibm1txD+OJ0qZNG0aPHl3cwxAIBPnAUJNuZWvLMJNWoKYiHaBi+/YA+NWrR9WuXQHzdPf//vmH7z09WdazJ+qsrMcxbIHgqcMg0r2Dg822pcfH53l8zIULbH7vPZJNjINLPCU0ki76owsETycFFumHDh3if//7X4H7ogsePzdv3mTYsGGUKVMGGxsbAgICGDVqFPdl/UxB1/N85syZxTPIx4xCoWDt2rX52nf16tW5tu8TCARPnqzUVE7OmUPi7dvSOo1ajSY7GwArOzs8KlfGLTBQ2m5JpLeeOJFR4eG8efo0rhUqAMYiPTkqirWDB6POyODymjUcfUq/EwWCoiZVL9J9atc225Yfkf7XCy9wYtYsNr3zTlEP7bGiUOlvmUtQTXpmTDSpF84C4Na6ffEORiAQFClPaeHQs0dYWBiNGjXi2rVr/Pvvv4SGhjJ79mzJLf/BgwfFMq6sEhidyszMBMDDwwNnZ+diHo1AIJCz67PP2PT22yxu//CG0xBFB1DZ2gJg5+4urbMk0lXW1rgFBqJQKKT020xZTXpsSIgk/AEiDxwouhchEDzFGFofelWvbrYtZPXqPI9PjY0FIGzHjqId2ONG6pNeciLpCft076FjnQZYe5n7dQgEgtKLEOlPCe+++y42NjZs376d1q1bU6FCBTp37szOnTu5ffs2n3/+OaBL8Y6IiOCDDz5AoVCgUCiMzrNt2zaCg4NxcnKiU6dO3L1712j7vHnzCA4Oxs7OjurVq/Pbb79J227cuIFCoWDZsmW0bt0aOzs7lixZYnG8CoWCOXPm0KVLFxwcHAgODubIkSOEhobSpk0bHB0dad68OdevXzc6bt26dTRo0AA7OzsqVqzIpEmTyNbfaAfqI2s9evRAoVBIz7/88kvq1avHvHnzCAoKws7OTnov5OnuGRkZjB07lvLly2Nra0vlypX5888/AYiLi2PAgAF4e3tjb29PlSpVWLBgQQH+hwQCQX648O+/ANwLCZHWZctEupX+71deR25JpMsxiHR5JD3NZOJSfr2CEB8RwT8vv0z47t2FOl4gKG0YSkOs7O2xsrc32rZv0iRuHjmS47HydrmaEpY2nheGmvSi6jdfFMTt3AKIVHeB4GlEiPQ80Gq1aDSaYnnkt/f7gwcP2LZtG//73/+wN/nB9PPzY8CAASxbtgytVsvq1a
```

Comment:
Are these intervals consistent with the new results from the reduced data? They should match the start and end times in the event window field for each event.